flycheck.el

```diff
@@ -8112,6 +8112,19 @@ for the `--crate-type' flag."
   :package-version '("flycheck" . "0.20"))
 (make-variable-buffer-local 'flycheck-rust-crate-type)
 
+(flycheck-def-option-var flycheck-rust-binary-name nil rust-cargo
+  "The name of the binary to pass to `cargo rustc --bin'.
+
+The value of this variable is a string denoting the name of the
+binary to build: either the name of the crate, or the name of one
+of the files under `src/bin'.
+
+This variable is needed only when `flycheck-rust-crate-type' is
+`bin' and there are multiple binary targets."
```

Comment:
Please rephrase this sentence a bit to emphasise that this option has only effect if `flycheck-rust-crate-type` is bin.